hoomd/md/EvaluatorPairDipole.h

```diff
@@ -288,17 +320,10 @@ class EvaluatorPairDipole
         Scalar rcutsq;              //!< Stored rcutsq from the constructor
         Scalar q_i, q_j;            //!< Stored particle charges
         Scalar4 quat_i,quat_j;      //!< Stored quaternion of ith and jth particle from constructor
-        const param_type &params;   //!< The pair potential parameters
+        Scalar mu;
+        Scalar A;
+        Scalar kappa;
```

Comment:
Is there a reason you switched from storing the params in the `param_type`? Either way works, but I am just wondering why the change.